lib/Partitioner/Partitioner.cpp

```diff
@@ -228,8 +228,8 @@ void Partitioner::initOpComputeTime(Function *F) {
     }
 
     // Repeat for outputs
-    if (node.getNumResults() > 0) {
-      auto myty = node.getType(0);
+    for (size_t i = 0, e = node.getNumResults(); i < e; i++) {
```

Comment:
random question: why only SparseLengthsWeightedSumNodeKind on line 195 and not fused versions etc.

Reply:
This was implemented by @nrsatish  and will be improved to handle more node type.